wagtail/core/tests/test_blocks.py

```diff
@@ -3275,6 +3275,26 @@ def test_deconstruct_target_model_multiple_literals(self):
             'wagtail.core.blocks.PageChooserBlock',
             (), {'page_type': ['tests.SimplePage', 'tests.EventPage']}))
 
+    def test_bulk_to_python(self):
+        page_ids = [2, 3, 4, 5]
+        expected_pages = Page.objects.filter(pk__in=page_ids)
+        block = blocks.PageChooserBlock()
+
+        with self.assertNumQueries(1):
+            pages = block.bulk_to_python(page_ids)
+
+        self.assertSequenceEqual(pages, expected_pages)
+
+    def test_bulk_to_python_when_child_of_listblock(self):
+        page_ids = [2, 3, 4, 5]
+        expected_pages = Page.objects.filter(pk__in=page_ids)
+        block = blocks.ListBlock(blocks.PageChooserBlock())
+
+        with self.assertNumQueries(1):
+            pages = block.to_python(page_ids)
+
+        self.assertSequenceEqual(pages, expected_pages)
```

Comment:
It is nice having these two tests together, as the patterns complement each other nicely. However, I think this last test would better belong to under a new `TestListBlock` class

Reply:
Agreed -- there is already a [`TestListBlock`](https://github.com/wagtail/wagtail/blob/a87b3fce9bf33fde53008d5987b28a22970cf22e/wagtail/core/tests/test_blocks.py#L1942) but I didn't add the test there because it's a SimpleTestCase and doesn't load the test.json fixture like TestPageChooserBlock does. I didn't want to convert it to a TestCase and thus add the database fixture loading unnecessarily to all of its tests. Maybe I can just add another TestListBlockWithFixture to isolate just this one test.

Reply:
> Maybe I can just add another TestListBlockWithFixture to isolate just this one test.

Yep, this sounds ideal :)